math-libs/BLAS/CMakeLists.txt

```diff
@@ -36,6 +36,58 @@ therock_cmake_subproject_provide_package(hipBLAS-common hipblas-common lib/cmake
 therock_cmake_subproject_activate(hipBLAS-common)
 list(APPEND _blas_subproject_names hipBLAS-common)
 
+
+##############################################################################
+# rocRoller
+##############################################################################
+
+if(WIN32)
+  set(_enable_rocRoller "OFF")
+else()
+  set(_enable_rocRoller "ON")
+endif()
```

Comment:
Please add a row to the Windows support table for this: https://github.com/ROCm/TheRock/blob/a2dd40ae7de646eb1d90edc27971de918280a9f6/docs/development/windows_support.md?plain=1#L55-L62